atest/testdata/keywords/optional_given_when_then.robot

```diff
@@ -98,5 +103,9 @@ We ${x} This ${thing} Implemented
 We Go To ${somewhere}
     Should Be Equal  ${somewhere}  walking tour
 
+${person} drinks ${beverage} instead
+    Should not start with  ${person}  Then
+    Should not start with  ${person}  and
```

Comment:
Validating the string with `Should Be Equal` would make the test more explicit and also a bit safer. The problem is then knowing who the `${person}` is, but perhaps there could just be two keywords `${person} drinks lonkero instead` and `${person} drinks water instead`. Not sure is that worth the effort, though.

Please use four spaces between arguments.

Reply:
Spacing updated to 4 spaces, also for existing code.

I did indeed go for the solution with two keywords. There are other options but they make it more complicated.